flask_app/controllers/players.py

```diff
@@ -0,0 +1,97 @@
+from flask import render_template, redirect, request, session
+from flask_app import app
+from flask_app.models.player import Player
+from flask_app.models.user import User
+
+
+from flask_bcrypt import Bcrypt 
+bcrypt = Bcrypt(app)
+
+@app.route('/create_link')
+def new_player():
+    if 'user_id' not in session:
+        return redirect('/')
+    return render_template('players_new.html')
+
+
+@app.route('/create', methods=["POST"])
+def new_players():
+    if 'user_id' not in session:
+        return redirect('/')
+    if not Player.validate_player(request.form):
+        return redirect('/create_link')
+    print(request.form)
+    data = {
+        "playername": request.form["playername"],
+        "playerteam" : request.form["playerteam"],
+        "playerposition": request.form["playerposition"],
+        "playerppg" : request.form["playerppg"],
+        "playerrpg": request.form["playerrpg"],
+        "playerapg" : request.form["playerapg"],
+        "playerspg": request.form["playerspg"],
+        "playerbpg" : request.form["playerbpg"],
+        'user_id': session['user_id']
+        }
+    
+    player_id = Player.create_a_player(data)
```

Comment:
what is the player_id variable for? We should be able to just keep it as "Player.create_a_player(data)" no?